Pipfile

```diff
@@ -16,9 +16,12 @@ django-cors-headers = "~=4.3.1"
 
 channels = "*"
 channels_redis = "*"
-daphne = "*"
```

Comment:
what is daphne?

Reply:
So `daphne` is an `ASGI` server used with Django Channels to handle WebSocket connections and other asynchronous features. Before this, we used `python manage.py server` to run the backend server, this command will run a `WSGI` server. By adding daphne, we allow the server to handle asynchronous communication and real-time data exchange with WebSocket